microcosm_pubsub/conventions/__init__.py

```diff
@@ -0,0 +1,34 @@
+"""
+Convention-driven pubsub.
+
+"""
+from functools import partial
+
+from microcosm_pubsub.conventions.lifecycle import LifecycleChange
+from microcosm_pubsub.conventions.messages import IdentityMessageSchema, URIMessageSchema  # noqa: F401
+from microcosm_pubsub.conventions.naming import make_media_type, name_for  # noqa: F401
+
+
+def media_type(lifecycle_change):
```

Comment:
This partial function can be used to generate new `verbed` wrappers. I intend to do away with `stopped` and move it to the one place our code base we use it as a custom lifecycle change.